services/app-api/forms/sar.json

```diff
@@ -593,30 +509,9 @@
                   "children": [
```

Comment:
```suggestion
  "children": [
    {
      "type": "text",
      "content": "In this section, provide information for the specified period."
    },
    {
      "type": "text",
      "content": "Active MFP participants excludes individuals whose enrollment in the MFP Demonstration ended because they completed their 365 days of MFP eligibility, died before they exhausted their 365-day enrollment period, were institutionalized for 30 days or more and did not subsequently re-enroll in the MFP program, or otherwise disenrolled from the program."
    }
  ]
```